src/Calculations/Max.py

```diff
@@ -0,0 +1,20 @@
+from src.Calculations import Calculation
+import numpy
+
+class Range(Calculation.Calculation):
+    def __init__(self):
+        super().__init__("Range", vertical=True, horizontal=True)
+
+    def do(self, data: [list]) -> list:
+        filtered_data = self.filter_out(data, str)
+        result = []
+        for rc in filtered_data:
+            if rc == []:
+                result.append(-1)
```

Comment:
in other code when rc == [], josh and I appended 0 instead of -1 (applies to this one, Min.py, Mode.py, Range.py)

Reply:
Actually I feel like neither makes sense, should we make it a string "None" instead?

Reply:
Let's do 0 for now (to maintain consistency with the calculations we already have) and we might change it later (let's discuss when we meet how we feel about this, I think we should change it too but we should all agree on what the best option is)